packages/api-server/controllers/auth.js

```diff
@@ -116,7 +118,7 @@ const login = async (req, res) => {
     }
     return res.json({
       token,
-      user: { id, email, name },
+      user: { id, email, fullname, address },
       message,
```

Comment:
```suggestion
      user: { id, email, fullname, address, status },
```
UNVERIFIED: no identity, user is unverified
WAITING: latest identity is unverified, user is unverified
VERIFIED: user is verified